Python 3 TabError in onionshare-nautilus.py

flake8 testing of https://github.com/micahflee/onionshare on Python 3.6.4

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__

./install/scripts/onionshare-nautilus.py:67:36: E999 TabError: inconsistent use of tabs and spaces in indentation
	file_uri = url.get_activation_uri()
                                   ^
1     E999 TabError: inconsistent use of tabs and spaces in indentation

--- a/install/scripts/onionshare-nautilus.py
+++ b/install/scripts/onionshare-nautilus.py
@@ -64,7 +64,7 @@ class OnionShareExtension(GObject.GObject, Nautilus.MenuProvider):
         """
 
     def url2path(self,url):
-	file_uri = url.get_activation_uri()
+        file_uri = url.get_activation_uri()
         arg_uri = file_uri[7:]
         path = urllib.url2pathname(arg_uri)
         return path

